com.microsoft.mrtk.spatialmanipulation/Solvers/TapToPlace.cs

```diff
@@ -517,16 +562,16 @@ private void UnregisterPickupAction()
         }
 
         /// <summary>
-        /// Extracts the InputAction from the InputActionReference.
+        /// Unregisters the event which performs placement.
         /// </summary>
-        /// <param name="actionReference">
-        /// The InputActionReference containing the desired InputAction.
-        /// </param>
-        /// <returns>An InputAction, or null.</returns>
-        public static InputAction GetInputActionFromReference(InputActionReference actionReference)
+        private void UnregisterPlacementEvent()
         {
-            if (actionReference == null) { return null; }
-            return actionReference.action;
+            if (interactable == null)
```

Comment:
since interactable is a required component, a null check doesn't make sense. This should be exceptional and cause an exception.